### ISSUE TITLE
Changes form master slide are not applied for web

### ISSUE THREAD

**Issue by yauheni-p** [OfficeDev/office-js#3197]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

# Provide required information needed to triage your issue

I try to change a text in a shape on a master slide with code:

PowerPoint.run(function(context) {
      var masterSlideShape = context.presentation.slideMasters.getItemAt(0).shapes.getItemAt(5);
      masterSlideShape .textFrame.textRange.text = 'Some text';
      return context.sync();
    });

This works fine with desktop PowerPoint. But when  it comes to web version, changes are not applied immediately. They are applied only after I refresh a page.

When I work with slides which are not master, everything is updated online.

No console errors

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: PC desktop, Office on the web
* Host [Excel, Word, PowerPoint, etc.]: PowerPoint
* Office version number: 1.1.84
* Operating System: Windows
* Browser (if using Office on the web): Chrome
 
## Expected behavior
Changes should be applied online

## Current behavior
Changes from master slide are applied only after page is reloaded

**Comment by alison-mk** [comment]
Hi @yauheni-p, thank you for submitting this issue! I've assigned it to the engineering team who will be able to provide you with more information or help resolve the issue. 

Cheers,
Alison
